gui/default/syncthing/folder/editFolderModalView.html

```diff
@@ -282,6 +285,9 @@
               <p translate class="help-block">
                 Disables comparing and syncing file permissions. Useful on systems with nonexistent or custom permissions (e.g. FAT, exFAT, Synology, Android).
               </p>
+              <p class="help-block" ng-if="currentFolder._recvEnc" translate translate-value-foldertype="{{friendlyNameFromFolderType(currentFolder.type)}}">
+                Always enabled when the folder type is "{%foldertype%}".
```

Comment:
This is a bit misleading.  For receive-encrypted, the option is always enabled, but it actually disables the function.  Use the other message or introduce a new "always disabled when..." variant?

Reply:
I've changed it to "turned on" as I think both "enabled" and "disabled" can be unclear here.